hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/mod/SubmitModificationsOp.java

```diff
@@ -39,12 +42,19 @@
  * original transaction.
  */
 public class SubmitModificationsOp extends UtilOp {
+    // An account to create as part of the submit-with-modifications process
+    // to ensure fees are charged (the default payer is a superuser account
+    // that is never charged fees); this ensures we cover calculateFees()
+    // code paths in handlers
+    private static final String MODIFIED_CIVILIAN_PAYER = "modifiedCivilianPayer";
```

Comment:
Unfortunately "civilian account" comment is also needed at `QueryModificationsOp`.  Sigh.  Don't change anything, but we really need that glossary.